Casks/font-microsoft-office.rb

```diff
@@ -1,38 +1,41 @@
 cask 'font-microsoft-office' do
-  version :latest
+  version '20170623'
   sha256 '249473568eba7a1e4f95498acba594e0f42e6581add4dead70c1dfb908a09423'
 
-  url 'https://fossies.org/windows/misc/PowerPointViewer.exe'
+  # business.uwm.edu/gdrive/Dietenberger_E/PowerPointViewer.exe was verified as official when first introduced to the cask
```

Comment:
@yurikoles Can you explain this comment? Where was this verified as official?

Reply:
It's just to make `brew cask audit` happy.